### PR TITLE
ddl: convert handle to string before assigning to Stat variables

### PR DESCRIPTION
**Pull request by tangenta**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/18109 

Problem Summary:

Only the basic types in Golang can be processed by `types.ToString`. `IntHandle` is not a basic type, so the error `'cannot convert 1(type kv.IntHandle) to string'` is reported.

### What is changed and how it works?

Convert the handle to string before assigning to Stat variables.

### Related changes

N/A

### Check List 

Tests 

- Unit test
- Integration test

Side effects

N/A

### Release note 

- No release note.

### CODE REVIEW

File: ddl/stat_test.go [hunk 1]
```diff
@@ -94,3 +96,60 @@ LOOP:
 		}
 	}
 }
+
+func (s *testStatSuite) TestDDLStatsInfo(c *C) {
```
Comment (wjhuang2016) [inline ddl/stat_test.go:100]: We have better use a serialSuite. In case the fail point affects other tests.

**Comment by coocood** [comment]
LGTM

**Pull request merged**